.github/workflows/shared-release-pull-request-library-java.yml

```diff
@@ -169,32 +157,9 @@ jobs:
           body: ${{ steps.release-notes-markdown.outputs.releaseNotes }}
           tag: ${{ needs.determine-version.outputs.version }}
 
-      - name: Create JIRA Release
-        id: create-jira-release
-        uses: cartoncloud/actions/jira-project-version-create@v3
-        with:
-          username: ${{ secrets.JIRA_USERNAME }}
-          password: ${{ secrets.JIRA_PASSWORD }}
-          serverAddress: ${{ vars.JIRA_SERVER }}
-          projectId: ${{ vars.JIRA_CARTONCLOUD_PROJECT_ID }}
-          name: ${{ inputs.repositoryName }} ${{ needs.determine-version.outputs.version }}
-          description: CartonCloud ${{ inputs.repositoryName }} ${{ needs.determine-version.outputs.version }}
-          isReleased: false
-
-      - name: Update JIRA fix versions
-        uses: cartoncloud/actions/jira-issues-update-fix-version@v3
-        with:
-          jiraServer: ${{ vars.JIRA_SERVER }}
-          jiraUsername: ${{ secrets.JIRA_USERNAME }}
-          jiraPassword: ${{ secrets.JIRA_PASSWORD }}
-          jiraReleaseId: ${{ steps.create-jira-release.outputs.releaseId }}
-
-      - name: Mark JIRA as released
-        uses: cartoncloud/actions/jira-project-version-update@v3
-        with:
-          username: ${{ secrets.JIRA_USERNAME }}
-          password: ${{ secrets.JIRA_PASSWORD }}
-          serverAddress: ${{ vars.JIRA_SERVER }}
-          projectId: ${{ vars.JIRA_CARTONCLOUD_PROJECT_ID }}
-          releaseId: ${{ steps.create-jira-release.outputs.releaseId }}
-          isReleased: true
+      - name: Add version comment
```

Comment:
Comment with the created version on merge. The version in the PR description is just a draft and may change if another PR is merged first so can't be trusted.